### PR TITLE
Bench 41: Add dependency management tool to project

### PR DESCRIPTION
**Pull request by MichaelOSullivanAnswer**
Setup poetry to manage dependencies and added packages from requirements.txt as well as black and pyright

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "answerking-python"
+version = "0.1.0"
+description = ""
+authors = ["michael-osullivan <michael.osullivan@answerdigital.com>"]
```
Comment (abbas-khan8) [inline pyproject.toml:5]: Not to steal your thunder but might be worth removing your name from the files

**Comment by PietroConvalleAD** [comment]
requirements.txt needs to be removed

File: pyproject.toml (hunk 1 above)
Comment (MichaelOSullivanAnswer) [reply pyproject.toml]: This is done

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+readme = "README.md"
+packages = [{include = "answerking_python"}]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+django = "^4.1.2"
+djangorestframework = "^3.14.0"
+django-environ = "^0.9.0"
+mysqlclient = "^2.1.1"
```
[the remark this answers is not in the document]
Comment (MichaelOSullivanAnswer) [reply pyproject.toml:14]: this is done

**Comment by MichaelOSullivanAnswer** [comment]
> requirements.txt needs to be removed

Done

**Review by abbas-khan8 — APPROVED** [review]

**Pull request merged**